test/utils/utils.ts

```diff
@@ -0,0 +1,66 @@
+import axios from "axios";
+import assert, { AssertionError } from "assert";
+import { existsSync, rmSync } from "fs";
+import shell from "shelljs";
+import { DEVNET_URL } from "../constants/constants";
+
+export function exec(cmd: string) {
+    const result = shell.exec(cmd);
+    assertEqual(result.code, 0, `Command ${cmd} failed.\n${result.stderr}`);
+
+    return result;
+}
+
+export function assertContains(output: string, pattern: string) {
+    if (!output.includes(pattern)) {
+        console.error("Pattern not in input");
```

Comment:
L16 can be removed and L17-18 can be placed inside a multiline string in L19

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/251/commits/a34e1106318964e889bb6e8e12a3829254a5a437